zerver/webhooks/jira/doc.md

```diff
@@ -14,8 +14,12 @@ Server versions 5.2 or greater. For older versions, you'll need our
 
 1. {!generate-webhook-url-basic.md!}
 
-1. Go to your Jira **Site administration** page, and click on **Jira**.
-   In the left sidebar, scroll down, and under **Advanced**, click
+1. Go to your Jira **Site administration** page. Click on the
+   **dropdown**( <i class="fa fa-ellipsis-h" aria-hidden="true"></i> )
```

Comment:
I'd call this a menu icon vs dropdown, and I wouldn't put it in bold. Also, looking at our help center documentation, we don't normally add the 'aria-hidden' attribute for these icons, so let's drop that here.